.github/workflows/images.yaml

```diff
@@ -38,17 +38,20 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-      - uses: docker://quay.io/cilium/image-maker:e55375ca5ccaea76dc15a0666d4f57ccd9ab89de
+      - uses: docker://quay.io/cilium/image-maker:1751527462-bb24228@sha256:b17984cf427f921c65409816b8890bbf893ea44a42c226a3a56aca90f3205939
         name: Register binfmt from multi-platform builds
+        env:
+          DOCKER_HUB_PUBLIC_ACCESS_ONLY: true
+          QUAY_PUBLIC_ACCESS_ONLY: true
```

Comment:
Note: this is necessary for similar reasons as #283, without those, this step fails with:
```
Unable to find image 'linuxkit/binfmt:a17941b47f5cb262638cfb49ffc59ac5ac2bf334' locally
docker: error getting credentials - err: exit status 1, out: `ANY_REGISTRY_USERNAME is not set`
```
(example [job](https://github.com/cilium/image-tools/actions/runs/16055095915/job/45364929126?pr=351))